Bottom: a0f8c8154762bdf192b899a8d94cf72db4eff8c8
Top:    9450fa1039b215c5da3c479b62228d72cbd5b18e
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-18 18:04:51 +0000

Mark segments as not round before falling back to line-line intersection

This avoids assert failures with debugging enabeled


---

diff --git a/src/polygon1.c b/src/polygon1.c
index 58ca3fa..32d0ef7 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -1292,6 +1292,8 @@ seg_in_seg_arc_arc (struct info *i, struct seg *s1, struct seg *s2)
       s1->v->radius == s2->v->radius)
     {
       printf ("Cop-out for co-circular arcs for now - they will be better handled by their line approximations. <<<<<<<<<<\n");
+      s1->v->is_round = false;
+      s2->v->is_round = false;
       return seg_in_seg_line_line (i, s1, s2);
     }
 #endif
